website/docs/r/network_interface_application_gateway_backend_address_pool_association.html.markdown

```diff
@@ -126,7 +126,7 @@ resource "azurerm_network_interface" "example" {
 resource "azurerm_network_interface_application_gateway_backend_address_pool_association" "example" {
   network_interface_id    = azurerm_network_interface.example.id
   ip_configuration_name   = "testconfiguration1"
-  backend_address_pool_id = azurerm_application_gateway.network.backend_address_pool[0].id
+  backend_address_pool_id = azurerm_application_gateway.network.id
```

Comment:
this needs to be the Backend Address Pool of the Application Gateway, rather than the Application Gateway itself?